backend/cmd/start.go

```diff
@@ -72,7 +72,8 @@ const (
 	flagName                  = "name"
 
 	// Postgres store
-	flagPGDSN = "pg-dsn"
+	flagPGDSN    = "pg-dsn"     // postgresql connection string
+	flagPGMaxTPS = "pg-max-tps" // postgresql maximum transactions per second cap
```

Comment:
I'm not sure how that would be done on postgresql, for potentially ephemeral clients, let me look into it.

Reply:
Now that I've read the rest of the PR more carefully, I think this should be an option on your new store rather than something that mentions Postgres?

My understanding is that your new store could potentially be independent of Postgres and that this TPS configuration is just to rate limit its writing to its backing store, whatever that is (Postgres or something else).

I suppose what I'm trying to say is that I found it confusing that the new option seems to configure some type of overall TPS cap with Postgres while that's not what it's doing.

Reply:
I'm up for selecting a better name to prevent confusion! Any suggestions?